webcompat/views.py

```diff
@@ -275,14 +276,19 @@ def create_issue():
 @app.route('/issues/<int:number>')
 @cache_policy(private=True, uri_max_age=0, must_revalidate=True)
 def show_issue(number):
+    issue_body_json, issue_body_status_code, issue_body_headers = api.proxy_issue(number)
+    issue_body_json = json.loads(issue_body_json)
+
+    issue_comments_json, issue_comments_status_code, issue_comments_headers = api.proxy_comments(number)
+    issue_comments_json = json.loads(issue_comments_json)
+
     """Route to display a single issue."""
```

Comment:
nit: The comment should be above, just after the `def`

Reply:
Thanks.